utils/watermark_disk_test.go

```diff
@@ -0,0 +1,87 @@
+package utils
+
+import (
+	"crypto/rand"
+	"os"
+	"testing"
+
+	"gotest.tools/assert"
```

Comment:
hmm most of the other tests here use `"github.com/stretchr/testify/assert"` looks like only providers/dell/dell_test.go is using `"gotest.tools/assert"` we should probably just use one only.

Reply:
+1 lets stick to `testify/assert`, the package in dell_test.go was most likely a mistake

Reply:
assert := `github.com/stretchr/testify/assert`